module4/homework/appendNewRecipe/AppendNewRecipe.cpp

```diff
@@ -0,0 +1,72 @@
+#include "AppendNewRecipe.hpp"
+
+#include <algorithm>
+#include <fstream>
+#include <iterator>
+#include <map>
+#include <sstream>
+
+constexpr char FILE_PATH[] = "recipes.txt";
+constexpr char INGREDIENTS_MESSAGE[] = "Skladniki:\n";
+constexpr char STEPS_MESSAGE[] = "\nKroki:\n";
```

Comment:
That kind of name convention is reserved for macros, try to avoid it :)

Reply:
Thanks! I didnt know about it ;p